.github/workflows/create-release.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types:
+      - completed
+
+permissions:
+  contents: write
+
+name: create-release
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Conventional Changelog Action
+        id: changelog
+        uses: TriPSs/conventional-changelog-action@v5
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          output-file: 'CHANGELOG.md'
+          git-user-name: 'github-actions[bot]'
+          git-user-email: 'github-actions[bot]@users.noreply.github.com'
+          version-file: './pyproject.toml'
```

Comment:
If you set `version-path` to `tool.poetry.version`, would it mean that we don't have to add a new `version` top-level field?

Reply:
I of course haven't tried this myself, just looking at the docs

Reply:
yes it does, just tried this out and it works a charm!